library/libraries.js

```diff
@@ -1088,5 +1103,50 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
             }
             return false;
         }
+    },
+    'SWFObject': {
+        icon: 'icon_48', // currently has no icon
+        url: 'https://github.com/swfobject/swfobject',
+        test: function(win) {
+            if (win.swfobject && win.swfobject.embedSWF) {
+                // 2.x - exact version only for 2.3
+                return { version: win.swfobject.version || null };
+            } else if(win.deconcept && win.deconcept.SWFObject) {
+                // 1.x
+                return { version: null };
+            }
+            return false;
+        }
+    },
+    'FlexSlider': {
+        icon: 'icon_48', // currently has no icon
+        url: 'https://woocommerce.com/flexslider/',
+        test: function(win) {
+            var jq = win.jQuery || win.$ || win.$jq || win.$j;
+            if (jq && jq.fn && jq.fn.jquery && jq.flexslider){
+                return { version: null };
+            }
+            return false;
+        }
+    },
+    'SPF': {
+        icon: 'icon_48', // currently has no icon
+        url: 'https://youtube.github.io/spfjs/',
+        test: function(win) {
+            if (win.spf && win.spf.init) {
```

Comment:
I checked versions `2.0.0`, `2.3.0` and `2.4.0` of SPF on cdnjs and `load`, `navigate` or `prefetch` should also work instead of `init`. But I didn't check any of the other versions. SPF doesn't seem to export version information.